### DISCUSSION TITLE
Question  on redirection

### DISCUSSION THREAD

**Discussion by FedericoSinisi** [drakkan/sftpgo#1561] [General]
Hi

I am getting started setting up the app using docker and I created a certificate using ACME. I was looking on the doc https://github.com/drakkan/sftpgo/blob/main/docs/full-configuration.md that is it possible to redirect the traffic from 80 to 443 automatically by enabling the following env variables: 

```
      SFTPGO_HTTPD__BINDINGS__0__SECURITY__ENABLED: true
      SFTPGO_HTTPD__BINDINGS__0__SECURITY__HTTPS_REDIRECT: true

```
But it seem not be working as expected. 

I am user 
```
~ $ sftpgo --version
SFTPGo 2.5.6-a6a92f0d-2023-12-18T17:47:53Z +metrics +azblob +gcs +s3 +bolt +mysql +pgsql +sqlite -unixcrypt +portable
```
 
And the config has the following :

```
        "security": {
          "enabled": false,
          "allowed_hosts": [],
          "allowed_hosts_are_regex": false,
          "hosts_proxy_headers": [],
          "https_redirect": false,
          "https_host": "",
          "https_proxy_headers": [],
          "sts_seconds": 0,
          "sts_include_subdomains": false,
          "sts_preload": false,
          "content_type_nosniff": false,
          "content_security_policy": "",
          "permissions_policy": "",
          "cross_origin_opener_policy": "",
          "expect_ct_header": ""
```

thanks!

**Comment by mombooks** [comment]
What browser/OS are you using? I'm finding http fails to redirect to https on Safari (both Mac and iPhone) but Firefox and Brave (Mac) redirect it just fine. Chrome and Edge on Windows 10 didn't redirect.

I configured using 

"https_redirect": true,

rather than the ENV vars which I can't see in the new docs, the config settings are in here
https://sftpgo.github.io/latest/config-file/#http-server

**Reply by drakkan** [reply]
@mombooks for details about redirect please that a look [here](https://github.com/drakkan/sftpgo/discussions/1170). Env vars are documented [here](https://sftpgo.github.io/latest/env-vars/), editing the json file is also an option if you prefer this way
